src/ontology/edits/oeo-physical.omn

```diff
@@ -1449,13 +1580,13 @@ The production of other coal gases (i.e. coke oven gas, blast furnace gas and ox
 Class: Generator
 
     Annotations: 
-        <http://purl.obolibrary.org/obo/IAO_0000115> "A generator is a grid component that converts motive power (mechanical energy) into electrical power for use in an external circuit.
-Source: https://en.wikipedia.org/wiki/Electric_generator"@en
+        <http://purl.obolibrary.org/obo/IAO_0000115> "A generator is an energy converting device that converts other forms of energy into electrical energy."@en,
+        <http://purl.obolibrary.org/obo/IAO_0000233> "issue: https://github.com/OpenEnergyPlatform/ontology/issues/173
+pull request: https://github.com/OpenEnergyPlatform/ontology/pull/273"
     
     SubClassOf: 
-        ArtificialObject,
-        uses some EnergyCarrierDisposition,
-        uses some EnergyGeneratorTechnology
+        <http://openenergy-platform.org/ontology/oeo-physical/EnergyConvertingDevice>,
+        has_physical_output some ElectricalEnergy
     
     DisjointWith: 
         Line
```

Comment:
Why `DisjointWith: Line`? Wouldn't it make more sense to state disjoint with `Turbine` and `Heater`?

Reply:
yes that's a relict

Reply:
changed it